algorithms/dp/bitmasking.py

```diff
@@ -93,7 +93,7 @@ def countUniqueWaysFrom(nbPeople, nbCaps, caps, DP, mask, cap_id):
 :param capSets: A nested list containing the caps of every person. 
 :returns: The nr of ways to select unique caps for every person.
 """
-def countUniqueWays(capsSets: list, nbCaps: int) -> int:
+def assign_unique_caps(capsSets: list, nbCaps: int) -> int:
```

Comment:
Could you also change the other function names to camel case in order to be consistent?

Reply:
Is not snake case the convention in python?

Reply:
I don't know, I'll have a look on it, and yes I will adjust the function name

Reply:
I just checked and it seems you're right, I'll update the code